pkg/client/disk.go

```diff
@@ -162,8 +168,8 @@ func (c *DiskServiceClient) DiskGet(diskType, diskName, diskPath string) (*api.D
 	}, nil
 }
 
-// DiskDelete deletes the disk with the given name and uuid.
-func (c *DiskServiceClient) DiskDelete(diskType, diskName, diskUUID string) error {
+// DiskDelete deletes the disk with the given name, uuid and path and driver.
```

Comment:
`// DiskDelete deletes the disk with the given name, uuid, path and driver.`

Reply:
Updated. Thank you.